src/Team-Services-Bot.Api/Web.config

```diff
@@ -14,7 +14,7 @@
   http://go.microsoft.com/fwlink/?LinkId=301879
   -->
 <configuration>
-  <appSettings configSource="AppSettings.config" />
+  <appSettings configSource="AppSettings.config.user" />
```

Comment:
Issue is that during runtime a user (contributor or viewer ) will miss the .user file. Which will stop them immediately. By adding the .user file, git will track the changes and we might by mistake commit it to the repo. 

I already tried these things with the appsettings.config file.

So could you please change it to AppSettings.config.